### ISSUE TITLE
Order status hover/order note missing (again)

### ISSUE THREAD

**Issue by codestylist** [woothemes/woocommerce#50434]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

After updating to WordPress 6.6.1 and WooCommerce 9.1.4, the tooltip with the order note is missing when I hover the mouse pointer over the order status button on the order page.

This problem already occurred two years ago with the #33228 (https://github.com/woocommerce/woocommerce/issues/33228). It was fixed, but now it's back again.

### Expected behavior

When I hover over the order status button on the orders screen the tooltip with the order note should appear.

![image](https://github.com/user-attachments/assets/5c18a514-ee06-48ab-9793-8fb62fc64531)

### Actual behavior

When I hover over the order status button on the orders screen the tooltip with the order note does not appear.

### Steps to reproduce

Update to WordPress 6.6.1 and WooCommerce 9.1.4.
Hover over the order status button on the orders screen.

### WordPress Environment

<details><summary>System Status Report</summary>

```
### WordPress Environment ###

WordPress address (URL): [Redacted]
Site address (URL): [Redacted]
WC Version: 9.1.2
Legacy REST API Package Version: The Legacy REST API plugin is not installed on this site.
Action Scheduler Version: ✔ 3.7.4
Log Directory Writable: ✔
WP Version: 6.6.1
WP Multisite: –
WP Memory Limit: 384 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: LiteSpeed
PHP Version: 7.4.33
PHP Post Max Size: 16 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.87.0
OpenSSL/1.1.1w

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.6.14-MariaDB-cll-lve-log
Max Upload Size: 16 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

[REDACTED]

### Post Type Counts ###

attachment: 12087
atum_supplier: 37
blocks: 16
mc4wp-form: 1
nav_menu_item: 76
nf_sub: 2534
oembed_cache: 60
page: 32
post: 36
pos_custom_product: 5
pos_grid: 1
pos_outlet: 2
pos_receipt: 2
pos_register: 4
pos_session: 1206
product: 3119
revision: 1079
shop_coupon: 3065
shop_order_placehold: 101270
user_request: 1
wc_order_email: 4
wc_order_status: 13
wpcf7_contact_form: 2
wp_global_styles: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (30) ###

Advanced Custom Fields: by WP Engine – 6.3.4
ATUM Inventory Management for WooCommerce: by Stock Management Labs™ – 1.9.39.2
Conditional Fields for Contact Form 7: by Jules Colle – 2.4.14
Classic Editor: by WordPress-Mitwirkende – 1.6.4
Code Profiler Pro: by Jerome Bruandet ~ NinTechNet Ltd. – 1.6.8
Honeypot for Contact Form 7: by WPExperts – 2.1.3
Contact Form 7: by Takayuki Miyoshi – 5.9.7
Disable Emails: by WebAware – 1.8.2
LiteSpeed Cache: by LiteSpeed Technologies – 6.3
MC4WP: Mailchimp for WordPress: by ibericode – 4.9.14
Product Sales Report for WooCommerce: by WP Zone – 1.5.6
Relevanssi: by Mikko Saari – 4.22.2
Remove Payment Gateway for Product: by Openstream Internet Solutions – 1.0.1
Advanced Order Export For WooCommerce: by AlgolPlus – 3.5.3
wallee: by wallee AG – 3.0.7
WC - APG Weight Shipping: by Art Project Group – 2.6.3
WooCommerce Cost of Goods: by SkyVerge – 2.13.1
Print Invoice & Delivery Notes for WooCommerce: by Tyche Softwares – 5.1.0
WooCommerce Google Merchant Center Feed: by claudiosanches – 1.0.6
WooCommerce Order Status Manager: by SkyVerge – 1.15.2
Pay for Payment for WooCommerce: by Karolína Vyskočilová – 2.1.8
Point of Sale for WooCommerce: by Actuality Extensions – 6.4.0
Sequential Order Numbers for WooCommerce: by SkyVerge – 1.10.1
WooCommerce: by Automattic – 9.1.2 (update to version 9.1.4 is available)
WooSidebars: by WooCommerce – 1.4.6
Yoast SEO Premium: by Team Yoast – 22.5
Yoast SEO: by Team Yoast – 23.1
Smush: by WPMU DEV – 3.16.5
WPS Hide Login: by WPServeur
NicolasKulka
wpformation – 1.9.16.7

YITH WooCommerce Wishlist: by YITH – 3.36.0

### Inactive Plugins (3) ###

GTM4WP - A Google Tag Manager (GTM) plugin for WordPress: by Thomas Geiger – 1.20.2
Swiss Post: by FirstPoint Sàrl – 2.2
WooCommerce Legacy REST API: by WooCommerce – 1.0.4

### Dropin Plugins () ###

advanced-cache.php: advanced-cache.php
object-cache.php: LiteSpeed Cache - Object Cache (Drop-in)

### Settings ###

API Enabled: –
Force SSL: ✔
Currency: CHF (CHF)
Currency Position: left_space
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: bundle (bundle)
external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-catalog (exclude-from-catalog-3)
exclude-from-catalog (exclude-from-catalog-5)
exclude-from-catalog (exclude-from-catalog-2)
exclude-from-catalog (exclude-from-catalog-4)
exclude-from-catalog (exclude-from-catalog-6)
exclude-from-search (exclude-from-search)
exclude-from-search (exclude-from-search-2)
featured (featured)
outofstock (outofstock-2)
outofstock (outofstock-3)
outofstock (outofstock)
rated-1 (rated-1-2)
rated-1 (rated-1-3)
rated-1 (rated-1)
rated-2 (rated-2-4)
rated-2 (rated-2-3)
rated-2 (rated-2)
rated-2 (rated-2-5)
rated-2 (rated-2-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5-3)
rated-5 (rated-5)
rated-5 (rated-5-2)

Connected to WooCommerce.com: ✔
Enforce Approved Product Download Directories: –
HPOS feature enabled: ✔
Order datastore: Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore
HPOS data sync enabled: –

### Logging ###

Enabled: ✔
Handler: Automattic\WooCommerce\Internal\Admin\Logging\LogHandlerFileV2
Retention period: 30 days
Level threshold: –
Log directory size: 35 KB

### WC Pages ###

Shop base: #10 - /shop/
Cart: #11 - /warenkorb/ -  Contains the [woocommerce_cart] shortcode
Checkout: #12 - /kasse/ -  Contains the [woocommerce_checkout] shortcode
My account: #13 - /mein-konto/
Terms and conditions: #151 - /agb/

### Theme ###

Name: Flatsome Child
Version: 3.5.0
Author URL: 
Child Theme: ✔
Parent Theme Name: Flatsome
Parent Theme Version: 3.19.4
Parent Theme Author URL: https://uxthemes.com
WooCommerce Support: ✔

### Templates ###

Overrides: flatsome/woocommerce/archive-product.php
flatsome/woocommerce/cart/cart-empty.php
flatsome/woocommerce/cart/cart-shipping.php
flatsome/woocommerce/cart/cart.php
flatsome/woocommerce/cart/mini-cart.php
flatsome/woocommerce/checkout/form-checkout.php
flatsome/woocommerce/checkout/form-coupon.php
flatsome/woocommerce/checkout/order-received.php
flatsome/woocommerce/checkout/terms.php
flatsome/woocommerce/checkout/thankyou.php
flatsome/woocommerce/content-product-cat.php
flatsome/woocommerce/content-product.php
flatsome/woocommerce/content-single-product.php
flatsome/woocommerce/content-widget-product.php
flatsome-child/woocommerce/emails/customer-completed-order.php
flatsome-child/woocommerce/emails/customer-on-hold-order.php
flatsome/woocommerce/global/breadcrumb.php
flatsome/woocommerce/global/quantity-input.php
flatsome/woocommerce/global/wrapper-end.php
flatsome/woocommerce/global/wrapper-start.php
flatsome/woocommerce/loop/header.php
flatsome/woocommerce/loop/loop-end.php
flatsome/woocommerce/loop/loop-start.php
flatsome/woocommerce/loop/pagination.php
flatsome/woocommerce/loop/rating.php
flatsome/woocommerce/loop/result-count.php
flatsome/woocommerce/loop/sale-flash.php
flatsome/woocommerce/myaccount/form-login.php
flatsome/woocommerce/myaccount/navigation.php
flatsome/woocommerce/notices/error.php
flatsome/woocommerce/notices/notice.php
flatsome/woocommerce/notices/success.php
flatsome/woocommerce/product-searchform.php
flatsome/woocommerce/single-product/photoswipe.php
flatsome/woocommerce/single-product/price.php
flatsome/woocommerce/single-product/product-image.php
flatsome/woocommerce/single-product/product-thumbnails.php
flatsome/woocommerce/single-product/rating.php
flatsome/woocommerce/single-product/related.php
flatsome/woocommerce/single-product/review.php
flatsome/woocommerce/single-product/sale-flash.php
flatsome/woocommerce/single-product/share.php
flatsome/woocommerce/single-product/short-description.php
flatsome/woocommerce/single-product/tabs/tabs.php
flatsome/woocommerce/single-product/title.php
flatsome/woocommerce/single-product/up-sells.php
flatsome/woocommerce/single-product-reviews.php
flatsome/woocommerce/single-product.php


### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customize-store
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
product-custom-fields
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page
launch-your-store

Disabled Features: experimental-blocks
minified-js
pattern-toolkit-full-composability
product-pre-publish-modal
printful
settings
async-product-editor-category-field
product-editor-template-system

Daily Cron: ✔ Next scheduled: 2024-05-04 12:21:37 +02:00
Options: ✔
Notes: 206
Onboarding: completed

### Action Scheduler ###

Complete: 3,083
Oldest: 2024-04-03 10:21:44 +0200
Newest: 2024-07-23 19:22:36 +0200

Pending: 7
Oldest: 2024-07-23 19:53:25 +0200
Newest: 2024-07-30 19:22:31 +0200

Failed: 626
Oldest: 2019-08-28 02:07:34 +0200
Newest: 2024-04-26 15:08:21 +0200

Canceled: 2
Oldest: 2024-04-24 20:47:51 +0200
Newest: -0001-11-30 00:34:08 +0034


### Status report information ###

Generated at: 2024-08-07 10:35:28 +02:00
```
</details>

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by naman03malhotra** [comment]
Closing it as a duplicate of https://github.com/woocommerce/woocommerce/issues/50284
